dist/pf-tabs.html

```diff
@@ -19,7 +19,7 @@
     </div>
 </template>
 
-<script>'use strict';
+<script src="pf-tabs.js">'use strict';
 
```

Comment:
I finally figured out what was going on with this... 

The vulcanize example I had followed was using an older version `gulp-vulcanize`. The new version accepts `inlineScripts` and `inlineCSS`, so to fix this, can you include this change in the PR:

```
gulp.task('vulcanize', ['html'], function () {
  return gulp.src('dist/pf-tabs.local.html')
    .pipe($.vulcanize({dest: 'dist', inlineScripts: true, inlineCss: true}))
```

This should inline any scripts and css correctly (as you can see, it's currently not).

Here's the other [options](https://github.com/Polymer/vulcanize#using-vulcanize-programmatically) available if interested.